CHANGES.rst

```diff
@@ -49,6 +49,9 @@ astropy.io.fits
 - Expose ``Header.strip`` as a public method, to remove the most common
   structural keywords. [#11174]
 
+- Use data from FITS record to construct mask when reading a table object from 
+  a FITS file. [#11372]
+
```

Comment:
Since this was working before #11222 and concerns only the dev version, we don't need a changelog. (Great to find this before the release!)

Reply:
Good call -- removed!